edx_solutions_api_integration/courses/tests.py

```diff
@@ -1275,6 +1271,37 @@ def test_courses_users_list_courses_passed(self):
         self.assertEqual(response.data["results"][0]["id"], users[0].id)
         self.assertEqual(response.data["results"][1]["id"], users[1].id)
 
+    def test_courses_passed_users_list_complete_status(self):
+        """ Test complete_status in users passed list of a course """
+        course = self.setup_course_with_grading()
+        test_uri = self.base_courses_uri + '/{course_id}/users/passed'
+
+        users = UserFactory.create_batch(2)
+        CourseEnrollmentFactory.create(user=users[0], course_id=course.id)
+        CourseEnrollmentFactory.create(user=users[1], course_id=course.id)
+
+        module = self.get_module_for_user(users[0], course, course.homework_assignment)
+        grade_dict = {'value': 1, 'max_value': 1, 'user_id': users[0].id}
+        module.system.publish(module, 'grade', grade_dict)
+        module = self.get_module_for_user(users[1], course, course.homework_assignment)
+        grade_dict = {'value': 1, 'max_value': 1, 'user_id': users[1].id}
+        module.system.publish(module, 'grade', grade_dict)
+
+        user_grade, user_proforma_grade = 0.9, 0.91
+        StudentGradebook.objects.update_or_create(
```

Comment:
I don't think we need to create gradebook here `module.system.publish(module, 'grade', grade_dict)` should do it. I mean we should eliminate either of two statements in this test.